bin/dev-bash

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+# DOC: Open a bash shell in the dev environment container.
+
+source bin/lib.sh
+
+bin/pull-image
+
+docker run -it --rm \
```

Comment:
i guess a lil nit if we are doing all the inputs on new lines maybe these should be new lined too?

Reply:
done

Reply:
done